_pages/schedule.json.liquid

```diff
@@ -0,0 +1,88 @@
+---
+permalink: /schedule.json
+layout: null
+---
+{
+  "schedule": {
+    "version": "0.1",
+    "base_url": "https://laborluxeria.github.io/winterchaos2022/schedule/",
+    "conference": {
+      "acronym": "winterchaos-2022",
+      "title": "Winterchaos",
+      "start": "2022-12-28",
+      "end": "2022-12-30",
+      "daysCount": 3,
+      "timeslot_duration": "00:05",
+      "time_zone_name": "Europe/Zurich"
+    },
+    "rooms": [
+      {
+        "name": "Winterchaos",
+        "guid": "99411f15-7335-4d36-abac-3ff0419ec954",
+        "description": "Talks at LuXerias and LABOR Luzerns Winterchaos 2022",
+        "capacity": null
+      }
+    ],
+    "days": [
```

Comment:
I quickly used the https://www.jsonschemavalidator.net/ on this schema I found: https://github.com/voc/schedule/blob/198e253cc2d673e7a29945c55ba9dd1e8be24935/validator/json/schema.json

It seems this `days` attribute needs to be a child of  `conference` above`. Other than that, it did not find any errors :partying_face: